modules/afsocket/afsocket-dest.c

```diff
@@ -323,6 +323,7 @@ _afsocket_dd_connection_in_progress(AFSocketDestDriver *self)
           msg_error("Syslog connection failed",
                     evt_tag_int("fd", self->fd),
                     evt_tag_str("server", g_sockaddr_format(self->dest_addr, buf, sizeof(buf), GSA_FULL)),
+                    evt_tag_str("group_name", self->super.super.group),
```

Comment:
why do we add the "group_name" here? It's not a usual practice to add it in similar messages. If we want to make it easier to locate this driver, maybe simply add the location in the config?

Reply:
I'll get rid of these as we will have the dest address stored across reload.

Reply:
I've kept `dest_addr` across reloads, so no need of this anymore.